README.md

```diff
@@ -40,148 +40,190 @@ For more information on the reliable web app pattern, see [Overview](https://rev
+g the above sequence.
+
+### 1. Clone the repo
+
+If using Windows, ensure you have enabled long paths.  Then clone the repository from GitHub:
 
-You can then enable support for long paths in git with the following command:
 ```shell
-git config --system core.longpaths true
+git clone https://github.com/Azure/reliable-web-app-pattern-java.git
+cd reliable-web-app-pattern-java
 ```
 
-If using WSL, start a WSL Ubuntu terminal and clone the repo to a WSL directory (see example in the following image).
+### 2. Open DevContainer in Visual Studio Code (optional)
+
+If required, ensure Docker Desktop is started.  Open the repository folder in Visual Studio Code.  You can do this from the command prompt:
 
-![WSL Ubuntu](docs/assets/wsl-ubuntu.png)
+```shell
+code .
+```
 
 Once Visual Studio Code is launched, you should see a popup allowing you to click on the button **Reopen in Container**.
 
-![WSL Ubuntu](docs/assets/vscode-reopen-in-container.png)
+![Reopen in Container](docs/assets/vscode-reopen-in-container.png)
 
-If you don't see the popup, open up the Visual Studio Code Command Palette. There are three ways to open the Code Command Palette:
+If you don't see the popup, open the Visual Studio Code Command Palette to execute the command. There are three ways to open the command palette:
 
 - For Mac users, use the keyboard shortcut ⇧⌘P
 - For Windows and Linux users, use Ctrl+Shift+P
-- In the VS Code, navigate to View -> Command Palette.
+- From the Visual Studio Code top menu, navigate to View -> Command Palette.
 
-Then, search for `Dev Containers: Rebuild and Reopen in Container` in the Command Palette.
+Once the command palette is open, search for `Dev Containers: Rebuild and Reopen in Container`.
 
 ![WSL Ubuntu](docs/assets/vscode-reopen-in-container-command.png)
 
-**2. Prepare for deployment**
+### 3. Log in to Azure
 
-Open a terminal in VS Code and enter the following. When prompted to enter a new environment name, choose one that's less than 18 characters.
+Before deploying, you must be authenticated to Azure and have the appropriate subscription selected.  To authenticate:
 
 ```shell
+az login --scope https://graph.microsoft.com//.default
 azd auth login
-azd config set alpha.terraform on
-azd env new
 ```
 
-Set the environment variables.
+Each command will open a browser allowing you to authenticate.  To list the subscriptions you have access to:
 
 ```shell
-azd env set DATABASE_PASSWORD <SOME_VALUE>
-azd env set AZURE_LOCATION <AZURE_REGION_NAME e.g. eastus>
-azd env set AZURE_SUBSCRIPTION_ID <AZURE_SUBSCRIPTION_ID>
+az account list
 ```
 
-> You can find a list of available Azure regions by running
-> the following Azure CLI command.
-> 
-> ```shell
-> az account list-locations --query "[].name" -o tsv
-> ```
+To set the active subscription:
 
-### (Optional) Multi-region support
+```shell
+az accout set --subscription "<your-subscription-id>"
```

Comment:
Found that when I applied copy and paste as recommended, I had to go back to the portal and get my subscriptionId again.

Would like to see "<your-subscription-id>" assigned to a variable or some other shorthand approach.

Reply:
Fixed.